tests/integration/009-api-update-loyalty-member-db.spec.ts

```diff
@@ -0,0 +1,196 @@
+import { expect, test } from '@playwright/test';
+import { PrismaService } from '~databases/prisma/dbService';
+import { registered_members } from '~databases/prisma/seeders/';
+import { envConfig } from '~src/configs/env';
+test.use({ baseURL: `http://${envConfig.HOST}:${envConfig.PORT}` });
+
+test.describe.configure({ mode: 'parallel' });
+test.describe('Update Loyalty Member Management', () => {
+	test.beforeAll(async () => {
+		console.log(test.info().workerIndex);
+		if (test.info().workerIndex === 1) {
```

Comment:
seems it doesn't work like we had expected. Mine got index of `50++` and it keeps increasing everytime it's failed (cause it's restarted) I will try another approach, if can't find any better solution may need to disable the parallelization

Reply:
can't find any better solution, disabled for now